ui/ducks/bridge/selectors.ts

```diff
@@ -108,17 +106,13 @@ export const getToTokens = (state: BridgeAppState) => {
   return state.bridge.toChainId ? state.metamask.bridgeState.destTokens : {};
 };
 
-export const getFromToken = (
-  state: BridgeAppState,
-): SwapsTokenObject | SwapsEthToken => {
+export const getFromToken = (state: BridgeAppState) => {
   return state.bridge.fromToken?.address
     ? state.bridge.fromToken
     : getSwapsDefaultToken(state);
 };
 
-export const getToToken = (
-  state: BridgeAppState,
-): SwapsTokenObject | SwapsEthToken | null => {
```

Comment:
Can we keep the response types here and in the above? Or perhaps what was the justification to remove them?

Reply:
I ran into type conflicts when using the token (it gets transformed to the `AssetWithDisplayData` in the AssetPicker component) so I removed this to let typescript infer the types. I fixed the conflicts though so I can put these types back